next.config.js

```diff
@@ -14,6 +14,7 @@ const REMOTE_PATTERNS = [
   // Temporary wildcard
   "*.s3.eu-west-1.amazonaws.com",
   "s3.eu-west-1.amazonaws.com",
+  "s3-alpha-sig.figma.com", //Added for Figma
```

Comment:
No need for a placeholder image so we can remove this. A person should be able to see an empty placeholder and then a preview when they upload.

Reply:
Ok will remove that as well